package.json

```diff
@@ -53,6 +53,7 @@
     "eslint-plugin-prettier": "3.0.1",
     "eslint-plugin-react": "7.13.0",
     "eslint-plugin-react-hooks": "1.6.0",
+    "gh-pages": "2.0.1",
```

Comment:
What's this been added for?

Reply:
The convenience of pre-publishing the demo. I suppose I could’ve used `npx`, but this aligns with our other repos. Lmk if you’d prefer to have it out and I’ll update.